packages/debian/changelog

```diff
@@ -1,5 +1,21 @@
 gdrcopy (@FULL_VERSION@) unstable; urgency=low
```

Comment:
remind me why we need unstable here

Reply:
Changed to stable. It is a legacy I forgot to change. Thanks for pointing it out.